README.md

```diff
@@ -1,60 +1,88 @@
+You will need the following software installed: 
+- Docker Desktop
+- VS Code
+- Remote Containers extension from the VS Code Marketplace
 
-## Automatic Setup
+## Bring up development environment
+To bring up the development environment you should clone this repo, change into the repo directory, and then open Visual Studio Code using the code . command. VS Code will prompt you to reopen in a container and you should select it. This will take a while the first time as it builds the Docker image and creates a container from it to develop in.
 
-The best way to use this repo is to start your own repo using it as a git template. To do this just press the green **Use this template** button in GitHub and this will become the source for your repository.
+```bash
+git clone git@github.com:Products-Development-Team/products.git
+cd products
+code .
+```
 
-## Manual Setup
+## Running the tests
+You can run the tests in a ```bash``` terminal using the following command: 
+```bash
+make test
+```
+This will run the test suite and report the code coverage. 
 
-You can also clone this repository and then copy and paste the starter code into your project repo folder on your local computer. Be careful not to copy over your own `README.md` file so be selective in what you copy.
+## Check PEP8 Standard
+We've included flake8, Pylint and Black in the ```requirements.txt```, you can check if the code is compliant using the following command: 
+```bash
+make lint
+```
 
-There are 4 hidden files that you will need to copy manually if you use the Mac Finder or Windows Explorer to copy files from this folder into your repo folder.
+## Run the REST service
+To run the service, use the same ```bash``` terminal that you ran the tests in and use 
+```bash
+honcho start
+``` 
+(Press CTRL+C to exit).
+You should be able to open a web page in a local browser at: http://localhost:8000
 
-These should be copied using a bash shell as follows:
+## Make REST calls
+While the service is running, you can open a second ``bash`` terminal and issue the following commands: 
 
+List all resources (Root URL):
+```bash
+http GET http://localhost:8000/
+```
+List all products: 
+```bash
+http GET http://localhost:8000/products
+```
+Create a product: 
 ```bash
-    cp .gitignore  ../<your_repo_folder>/
-    cp .flaskenv ../<your_repo_folder>/
-    cp .gitattributes ../<your_repo_folder>/
+http POST localhost:8000/products name="" description="" category="" price:=<float> available:=<bool> rating:=<int>
 ```
+You must specifiy the ``name``, ``description``, ``price`` and ``rating``of the product. 
```

Comment:
we haven' t included acceptable name and description yet (any name/description is acceptable ) yet so I think for now, is best to remove line 63 and 64. Otherwise everything looks good. Thank You!

Reply:
Got it